docs/pages/build/zero-to-one/registration-and-deployment.mdx

```diff
@@ -0,0 +1,15 @@
+# Registering and deploying a Micro-rollup
+
+Once you are done with the setup, development and sandbox testing of your micro-rollup, it is time to register and deploy your application on Stackr's network and on-chain.
+
+## Registering your Micro-rollup
+
+Refer to [`register`](/build/cli/register) command in CLI section.
+
+## Deploying your Micro-Rollup
+
+Refer to [`deploy`](/build/cli/deploy) command in CLI section.
+
+## Conclusion
+
+In this guide, we learned how to create a micro-rollup using the Stackr SDK, deploy it to the Vulcan network, and interact with it using the SDK.
```

Comment:
```suggestion
In this guide, we learned how to create a micro-rollup using the Stackr SDK, register and deploy it to the Vulcan network, and interact with it using the SDK.
```

and then like to the Framework and Plugin section for more advanced use-cases and understanding the tooling to leverage it in a better way